Applications/YOLOv3/readme.md

```diff
@@ -0,0 +1,41 @@
+# Getting started
+
+## Installation and build method
+Please refer to the https://github.com/nnstreamer/nntrainer/blob/main/docs/getting-started.md link for the NNTrainer installation and meson build.
+
+## Setting dataset and code modification
+Folders containing training data must include "images" and "annotations" folders.
+We supports BMP format for image, and TXT format for annotation.
+In addition, image files should be prepared in the same size.
+
+```bash
+...
+├─── train_dir
+│   └─── images
+│      └─── 001.bmp
+│      └─── 002.bmp
+│           ...
+│   └─── annotations
+│      └─── 001.txt
+│      └─── 002.txt
+│           ...
+...
+```
+
+Annotation data format is "label x_pos y_pos width height".
```

Comment:
quick question. what does `x_pos` and `y_pos` indicate?
1. x_centor/y_centor
2. x_start/y_start
3. x_end/y_end